version.txt

```diff
@@ -1 +1 @@
-1.15.0-dev
+1.16.0-dev
```

Comment:
I assume the file/examples in this PR will be usable only after images are built which would be 30mins after merge or so?

Reply:
Yes, that is the case after beginning of every new minor version dev.